client/games/game-players-display.tsx

```diff
@@ -3,38 +3,46 @@ import React, { useCallback, useMemo } from 'react'
 import { useTranslation } from 'react-i18next'
 import styled from 'styled-components'
 import { ReadonlyDeep } from 'type-fest'
+import { GameConfigPlayer } from '../../common/games/configuration'
 import { GameRecordJson } from '../../common/games/games'
 import { RaceChar } from '../../common/races'
 import { SbUser } from '../../common/users/sb-user'
+import { SbUserId } from '../../common/users/sb-user-id'
 import { RaceIcon } from '../lobbies/race-icon'
 import { useAppSelector } from '../redux-hooks'
 import { RootState } from '../root-reducer'
 import { labelMedium, singleLine, titleSmall } from '../styles/typography'
 
 const GamePreviewPlayers = styled.div`
-  column-count: 2;
-  column-gap: 16px;
-  text-align: start;
+  display: flex;
```

Comment:
I think the flex change is fine, but FWIW the things you "fixed" by moving to flex work fine with grid too. You could have used `row-gap` instead of the bottom margin, and could have explicitly assigned the players to specific columns instead of adding blank elements

Reply:
I actually did try using grid for this initially, while all the players were in a single array, and I couldn't get it to fill columns first. Which I later learned is not possible with grids :d:
https://stackoverflow.com/questions/44092529/make-grid-container-fill-columns-not-rows

I did think that it actually might be possible to do that with pre-defined grid areas and then assigning players to each grid area, but honestly, using flexbox seemed more straightforward to me and a bit easier to reason about to have a separate element for each team.

If you want, I can try reworking this with grids.

Reply:
Yes, I was talking about either using predfined grid areas with 1 area per column, or explicitly setting the 1-column ranges on the player elements. I don't really care for this particular PR, you can leave it as-is, I was just saying for future reference